base/context.go

```diff
@@ -113,18 +113,14 @@ func (ctx *Context) GetClientTLSConfig() (*tls.Config, error) {
 	}
 
 	if ctx.Certs != "" {
-		if log.V(1) {
-			log.Infof("setting up TLS from certificates directory: %s", ctx.Certs)
-		}
+		log.Printf("setting up TLS from certificates directory: %s\n", ctx.Certs)
```

Comment:
Rather than changing the `base` package, I wonder if it would be better to remove the dependency on `base` from `sql/driver`. The only thing `sql/driver` is using from `base` is `base.Context`. And the only method on `base.Context` is `GetHTTPClient`.